### ISSUE TITLE
Paste notifications not read by screen reader

### ISSUE THREAD

**Issue by mlewand** [ckeditor/ckeditor-dev#420]
## Are you reporting a feature or a bug?

Bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)

http://dev.ckeditor.com/ticket/16962 - contains some useful information

## Provide detailed reproduction steps (if any)

1. Open http://nightly.ckeditor.com/17-04-11-06-09/standard/samples/
2. Press "Paste" button.
3. Check if screen reader is announcing notification correctly.

### Expected result

Screen reader reads the notification.

### Actual result

* Firefox + VoiceOver: screen reader announces "Firefox has new window", but reads notification only after clicking it.
* Firefox + JAWS: screen reader becomes muted when notification appears.
* Chrome + VoiceOver: notification is read.

**Comment by mlewand** [comment]
This is an upstream issue. The problem is that JAWS will not read `div[role=alert]` nested in a regular element. For instance it works better with Chrome, but doesn't on Firefox.

We created a simplified reproduce case:

1. Open https://codepen.io/mlewand/pen/eRgypE?editors=1010#0
1. Focus "click me" button.
1. Press `space` key.

## Expected

Jaws reads: "Alert, Hello world"

## Actual

JAWS + Firefox @ Win10 - Doesn't work.
JAWS + Chrome @ Win10 - Reads the inner text, but does not say it's an alert.

## Other Details

JAWS 18.0.2945
Firefox 5.0 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:54.0) Gecko/20100101 Firefox/54.0
Chrome: 5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.36


NVDA + Firefox @ Win10 - doesnt work either.
NVDA + Chrome @ Win10 - works as expected.

Microsoft Narrator + Chrome @ Win10 - doesnt work either.
Microsoft Narrator + Firefox @ Win10 - Reads the inner text, but does not say it's an alert.

I have reported this issue to Freedom Scientific - as they do not provide any way to publicly track the issue I'll keep this issue updated if I get any feedback.
